_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -39,15 +40,20 @@ Each test target's `autocomplete` [attribute value][] is a [space separated][] l
 
 ## Expectation 2
 
-Each test target's `autocomplete` [attribute value][] has a [correct autocomplete field][] that is [appropriate][appropriate field for the form control] for that test target.
+Each test target's `autocomplete` [attribute value][] has a [correct autocomplete field][] that is [appropriate][appropriate field for the form control] for that its control group. It means that, for example, an `input` element with `autocomplete="new-password"` must have the `type` attribute with one of the following values:
+
+- `hidden` (in this case the test target is inapplicable),
+- `text`,
+- `search`,
+- `password`.
 
 ## Assumptions
 
 The `autocomplete` attribute is used on form fields that correspond to [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG21/#input-purposes) and collect information about the user.
 
-If the `autocomplete` attribute is used to describe "custom" taxonomy, for example `<input type="text" autocomplete="banner" />`, success Criterion [1.3.5 Identify Input Purpose][sc135] may be satisfied even if this rule failed.
+Controls with the `autocomplete` attribute that wears the [autofill expectation mantle](https://html.spec.whatwg.org/#autofill-expectation-mantle) (e.g. `<input autocomplete="off" />`) are assumed not to collect information about the user or do not correspond to [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG21/#input-purposes); therefore, they are inapplicable.
```

Comment:
That seems to directly contradict the specs:
> When wearing the autofill expectation mantle, the autocomplete attribute describes what input is expected from users.

Reply:
I made this assumption based on:

> The "off" keyword indicates either that the control's input data is particularly sensitive (for example the activation code for a nuclear weapon); or that it is a value that will never be reused (for example a one-time-key for a bank login) and the user will therefore have to explicitly enter the data each time, instead of being able to rely on the UA to prefill the value for them; or that the document provides its own autocomplete mechanism and does not want the user agent to provide autocompletion values.

To be honest, after reading it again, I'm not sure why I did it. Will be removed. Thanks!